dll/win32/browseui/shellfind/CFindFolder.cpp

```diff
@@ -809,11 +809,10 @@ class CFindFolderContextMenu :
                 CComHeapPtr<ITEMIDLIST> folderPidl(ILCreateFromPathW(_ILGetPath(apidl[i])));
                 if (!folderPidl)
                     return E_OUTOFMEMORY;
-                CComHeapPtr<ITEMIDLIST> filePidl(ILCombine(folderPidl, _ILGetFSPidl(apidl[i])));
-                if (!filePidl)
-                    return E_OUTOFMEMORY;
-                SHOpenFolderAndSelectItems(folderPidl, 1, &filePidl, 0);
+                LPCITEMIDLIST child = _ILGetFSPidl(apidl[i]);
+                SHOpenFolderAndSelectItems(folderPidl, 1, &child, 0);
             }
+            LocalFree(apidl); // Yes, LocalFree
```

Comment:
What's with the comment?

Reply:
That's why a more complete comment would be better (basically, what you explained to learn-more, but in the comment).